trustme/__init__.py

```diff
@@ -419,6 +420,36 @@ def configure_trust(self, ctx):
                 "unrecognized context type {!r}"
                 .format(ctx.__class__.__name__))
 
+    @classmethod
+    def from_pem(cls, cert_path, private_key_path):
+        """Build a CA from existing cert and private key.
+
+        This is useful if your test suite has an existing certificate authority and
+        you're not ready to switch completely to trustme just yet.
+
+        Args:
+          cert_path (str): The path to the certificate in PEM format
+          private_key_path (str): The path to the private key in PEM format
+
+        Returns:
+          CA: the newly-generated certificate authority
+        """
```

Comment:
Does this show up in the docs automatically, or do we need to add it?

Reply:
Do you mean the `Returns` part? It shows up like this in the docs:

<img width="514" alt="Capture d’écran 2019-10-31 à 08 48 01" src="https://user-images.githubusercontent.com/42327/67919813-53821180-fbbb-11e9-932f-1dcd77129559.png">

Without it, this part disappears

Reply:
Oh no, I meant the whole docstring :-). That answers the question though!